getting-started/windows-calico/quickstart.md

```diff
@@ -106,6 +106,10 @@ The following steps install a Kubernetes cluster on a single Windows node, with
    c:\install-calico-windows.ps1 -KubeVersion <your Kubernetes version (e.g. 1.18.6)> \
                                  -Datastore etcdv3
                                  -EtcdEndpoints <your etcd endpoint ip>
+                                 -EtcdTlsBySecret:$true (default not downloading etcd TLS files from calico-etcd-secrets secret)
+                                 -EtcdKey <path to key file> (default not using TLS)
+                                 -EtcdCert <path to cert file> (default not using TLS)
+                                 -EtcdCaCert <path to ca cert file> (default not using TLS)
```

Comment:
Could you add Etcd related items to #### Configure installation parameters below?